### DISCUSSION TITLE
Groq API and LMstudio API

### DISCUSSION THREAD

**Discussion by CGAlei** [longy2k/obsidian-bmo-chatbot#80] [General]
Hi longy2k, 

It would be fantastic if you could update it to incorporate Groq API and LMstudio API. Having this LLM in the profiles would be great. Your plugin ranks as one of the most comprehensive and efficient AI plugins for Obsidian. I absolutely adore the ease of use and the RAG capabilities.

**Comment by longy2k** [comment]
Currently the Groq API and LM Studio connection can be added to the REST API Connection one at a time. I might add more dedicated API connections over time.

Thank you for your feedback ☺️

**Reply by 3-ark** [reply]
Groq API is different from openai compatible API, it has a key as well. If Cohere API to be added?
